easybuild/easyconfigs/g/GREASY/GREASY-2.1-cscs-CrayGNU-18.07-gpu.eb

```diff
@@ -0,0 +1,24 @@
+easyblock = 'ConfigureMake'
+
+name = 'GREASY'
+version = '2.1-cscs'
+versionsuffix = '-gpu'
+
+homepage = 'https://github.com/jonarbo/GREASY'
+description = """Greasy is a tool designed to make easier the deployment of
+embarrassingly parallel simulations in any environment. It is able to run in
+parallel a list of different tasks."""
+
+toolchain = {'name': 'CrayGNU', 'version': '18.07'}
+toolchainopts = {'opt': True, 'pic': True}
+
+sources = ['/apps/common/easybuild/sources/g/GREASY/greasy-%(version)s-gpu.tar.gz']
```

Comment:
If you don't use sources from a git repo, could you please add as comment where does this version come from? (same for non-gpu version). If you modified the sources, maybe you should fetch it from your repository?

Reply:
https://github.com/eth-cscs/production/pull/692/commits/f768b898eaea3e15e04fcd5a762b679e65e5d82d

Reply:
done

Reply:
fixes https://github.com/eth-cscs/production/pull/692#pullrequestreview-143877649